vtex-apps/checkout-cartman
# Initial version for Cartman

#### What problem is this solving?

This PR implements (and releases) the initial version for this app. This PR is just a collection of previously approved PRs (#1 and #2).

#### How should this be manually tested?
This app is linked in [this workspace](https://lucas--checkoutio.myvtex.com/cart), but you need the "Call Center Operator" permission to see it (you can give yourself the permission in the admin by going into Account management > Access profiles > Call center operator and typing in your email, you may also need to wait for a while and clear cookies/localStorage).

#### Checklist/Reminders


#### Type of changes


✔️ | Type of Change
---|---
_ | Bug fix 
✔️ | New feature 
_ | Breaking change 
_ | Technical improvements
